src/panel/category/CategoryPanel.jsx

```diff
@@ -45,11 +44,22 @@ export default class CategoryPanel extends React.Component {
     window.execOnMapLoaded(() => { this.fitMapAndFetch(); });
   }
 
-  componentDidUpdate() {
+  componentDidUpdate(prevProps) {
+    const { bbox } = this.props;
+    const { category } = this.props.poiFilters;
+
     const panelContent = document.querySelector('.panel-content');
     if (panelContent) {
       panelContent.scrollTop = 0;
     }
+
+    if (category && category !== prevProps.poiFilters.category) {
```

Comment:
If there is no category in the new props (so `category` is null), we never pass this test. This means if there was an active category before, the points won't be refreshed…
To anticipate for `query` and other fields of `poiFilters`, it's probably safer to compare `poiFilters` as a whole.

Reply:
Ok I update by comparing if any key has changed instead